mandatory/2-function-creation.js

```diff
@@ -6,86 +6,130 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+    if (num <= 100 && num % 2 === 0) {
+        return true;
+    }
+
+    return false;
+};
+
 
 /*
 Write a function that:
-- takes a number as input
-- return a string formatted as percentages (e.g. 10 => "10%")
-- the number must be rounded to 2 decimal places
-- numbers greater 100 must be replaced with 100
+takes a number as input
+return a string formatted as percentages (e.g. 10 => "10%")
+the number must be rounded to 2 decimal places
+numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+    if (num > 100) {
+        return 100 + "%";
+    }
+    if (Number.isInteger(num)) {
+        return num + "%";
+    }
+
+
+    var rounding = Math.round(num);
+    var n = num.toFixed(2);
+    var numberPlusePrecentage = n + '%';
+    var twodicemalpalce = numberPlusePrecentage;
+    return numberPlusePrecentage;
```

Comment:
The logic here is good, the only thing I'd say is make sure you use either `let` or `const` instead of `var` because it's outdated. Also, as an alternative, you could use `Math.round(num * 100) / 100`, this way you don't need an `if` block for when the number is an integer, cause this works on it too https://stackoverflow.com/questions/11832914/how-to-round-to-at-most-2-decimal-places-if-necessary

Reply:
Also you could get rid of the `rounding` variable because it's not used anywhere 🙂